docs/components/compact.md

```diff
@@ -66,7 +66,8 @@ compacting blocks from an instance even when a Prometheus instance goes down for
 
 ## Flags
 
-[embedmd]:# (flags/compact.txt $)
+[embedmd]: # "flags/compact.txt $"
```

Comment:
Can we also add something about deletions?

Reply:
(e.g how blocks are deleted)

Reply:
will add that after we finalise where to upload the file, whether it should be in block or outside block